app/models/manageiq/providers/hawkular/middleware_manager/live_metrics_capture.rb

```diff
@@ -98,5 +95,13 @@ def process_data(metric, data)
         processed.store_path(timestamp, metric[:name], value)
       end
     end
+
+    private
+
+    def extract_feed(ems_ref)
```

Comment:
This method is private.
No unittest itself, it is part of metrics_available method which is tested on middleware_server_spec.rb or middleware_datasource_spec.rb.